README.md

```diff
@@ -1,4 +1,163 @@
 urg_node
 ===================
 
-ROS wrapper for the Hokuyo urg_c library.
\ No newline at end of file
+ROS 2 wrapper for the Hokuyo urg_c library.
+
+## Build instructions
+Install ROS 2 as per instructions [here](https://index.ros.org/doc/ros2/Installation/#installationguide)
+
+Create a folder for urg and clone this repo
+
+```
+mkdir -p ~/ros2_ws/src/urg
+cd ~/ros2_ws/src/urg
+git clone -b ros2-devel https://github.com/hopetechnik/urg_node.git
+```
+Clone dependencies, you might need to install the vcs command to do so :
+```
+sudo apt install python3-vcstool
+```
+then clone :
+```
+cd ~/ros2_ws/src/urg/urg_node
+vcs import ../ < urg.repos
+```
```

Comment:
with the set of PRs open (and ideally released) these instructions should be not necessary in the first place.

Reply:
Sure, makes sense to me.  Let's just remove this bit then.